x/init.go

```diff
@@ -64,3 +67,13 @@ To say hi to the community       , visit https://dgraph.slack.com.
 		os.Exit(0)
 	}
 }
+
+// Printf does a log.Printf. We often do printf for debugging but has to keep
+// adding import "fmt" or "log" and removing them after we are done.
+// Let's add Printf to "x" and include "x" almost everywhere. Caution: Do remember
+// to call x.Init. For tests, you need a TestMain that calls x.Init.
+func Printf(format string, args ...interface{}) {
```

Comment:
Thanks for the suggestion. I agree we should move towards using one single log function. However, x.Trace needs a ctx object and a lot of our functions don't have that yet.... And sometimes, for long running processes in the background, we use something else. So, I am not sure what is the best long term solution here.. For now, let me just use this one? We can do that in a separate PR..